plugins/ipam/dhcp/dhcp_test.go

```diff
@@ -331,3 +339,267 @@ var _ = Describe("DHCP Operations", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 })
+
+const (
+	hostBridgeName string = "dhcpbr0"
+	hostVethName0  string = "br-eth0"
+	contVethName0  string = "eth0"
+	hostVethName1  string = "br-eth1"
+	contVethName1  string = "eth1"
+)
+
+var _ = Describe("DHCP Lease Unavailable Operations", func() {
+	var originalNS, targetNS ns.NetNS
+	var dhcpServerStopCh chan bool
+	var dhcpServerDone *sync.WaitGroup
+	var clientCmd *exec.Cmd
+	var socketPath string
+	var tmpDir string
+	var err error
+
+	var br *netlink.Bridge
+
+	BeforeEach(func() {
+		dhcpServerStopCh = make(chan bool)
```

Comment:
The next 100+ lines look very similar to lines 49 onwards in dhcp2_test.go - if they are different please add some comments indicating what is going on, otherwise pull out the common code to a shared setup function.

Reply:
I created a shared setup function as suggested